lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,181 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = ManageIQ::Providers::IbmCloud::Engine.root.join("spec")
+  cass_dir = base_dir.join("vcr_cassettes/manageiq/providers/ibm_cloud")
+  spec_dir = base_dir.join("models/manageiq/providers/ibm_cloud")
+
+  # Prerequisites:
+  # 1. Log in with IBM Cloud CLI i.e. `ibmcloud login -sso`
+  # 2. Place IBM Cloud API key in secrets config
+  desc "Generate refresher VCR"
+  task :refresh_init => :environment do
+    cass_dir.glob("vpc/cloud_manager/**/*.yml").each(&:delete)
+
+    begin
+      apikey = YAML.load_file("config/secrets.yml").dig("test", "ibm_cloud_vpc", "api_key")
+    rescue => NoMethodError
+      raise "IBM Cloud API key not found in secrets config file"
+    end
+    token = IBMCloudSdkCore::IAMTokenManager.new(:apikey => apikey).access_token
+    auth = IBMCloudSdkCore::BearerTokenAuthenticator.new(:bearer_token => token)
+
+    connection = IbmVpc::VpcV1.new(:authenticator => auth)
+    resource_controller = IbmCloudResourceController::ResourceControllerV2.new({:authenticator => auth})
+
+    # Create resources
+    begin
+      default_resource_group_id = `ibmcloud resource group default --id | tr -d '\n'`
+      cloud_db_id = resource_controller.create_resource_instance(
+        :name             => 'rake-db',
+        :target           => 'bluemix-us-south',
+        :resource_group   => default_resource_group_id,
+        :resource_plan_id => 'databases-for-postgresql-standard'
+      ).result['id']
+      status = resource_controller.get_resource_instance(:id => cloud_db_id).result['state']
+      puts "Provisioning cloud database..."
+      until status == 'active'
+        sleep(10)
+        status = resource_controller.get_resource_instance(:id => cloud_db_id).result['state']
+      end
+      puts "Provisioned cloud database"
```

Comment:
I've added these helpful put statements here since it can take a while to provision a cloud database